charts/posit-chronicle/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: posit-chronicle
 description: Official Helm chart for Posit Chronicle Server
-version: 0.3.0
-appVersion: 2024.03.0
+version: 0.4.0
```

Comment:
I think this should be `0.3.1`. Other product version bumps just increment the patch version for the chart. (For example: https://github.com/rstudio/helm/pull/495/files)

Reply:
Sounds good!